processor/resourcedetectionprocessor/internal/dynatrace/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+type: resourcedetectionprocessor/dynatrace
```

Comment:
I think we need to add a codeowners entry somewhere in this file to fix the presubmit

Reply:
the other detectors seem to not have any codeowner entries, so i"m not sure where to add this - however it seems like the check just compares the codeowners file of the main branch vs the branch of the PR and fails if they differ, so I think after merging this will be resolved?

Reply:
Just add:

```yaml
status:
  codeowners:
    active: [bacherfl]
```

Reply:
it seems like this is not possible for sub components, as i got the following error message:

```
Error: failed loading /home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/dynatrace/metadata.yaml: status must be empty for subcomponents
```

Reply:
I have now removed the `parent` property from the metadata file. This seems to be the only way of having a code owner entry for a sub component. This is e.g. also done for the different scrapers of the hostmetrics receiver: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/metadata.yaml

Reply:
it seems like there is also a stability level required for the metadata.yaml, as I had to remove the `parent` - should we use the same stability level as the resourcedetection processor itself here, or set it to `development`/`alpha` as this is a new sub component?

Reply:
@dashpole I was able to add Florian and I as codeowners for this directory, and have a GH label created for it as well so you can easily ping us on issues.

mdatagen doesn't support this use case, so it isn't a perfect fit, but I think it is close enough. Let us know if you're alright with the setup now.

Reply:
Actually, thinking on it, the name is probably way too long for a label. We could also drop mdatagen and just list ourselves in the readme for you to manually ping us, I'm okay with whatever you prefer.

Reply:
whatever works is fine with me. If it isn't possible, then we can skip having codeowners

Reply:
Sounds good, thanks. I'll go forward with this for now, and we can just adjust course if needed since it will be easy to do.